concepticondata/conceptlists/Verheyen-2019-1000.tsv

```diff
@@ -0,0 +1,1001 @@
+Verheyen-2019-1000-343	343	hecht	solid			8.8	5	3.73684210526316	4.25	3.55	5.1	5.15	5	1	31776	2	555	191	0.180889066904373	4.32318145542922E-05
+Verheyen-2019-1000-344	344	hedendaags	contemporary			9.85	3.95	3.84210526315789	4.2	3.95	4.95	5.3	10	3	97277	0	707	93	0.550647265744071	8.40327547787345E-05
+Verheyen-2019-1000-345	345	heel	intact	INTACT	220	4.6	4.55	3.1	4.5	6.05	3.5	4.65	4	1	38598	22	23699	72007	0.788090316033179	0.000191293864265
+Verheyen-2019-1000-346	346	heerlijk	beautiful	BEAUTIFUL	1427	6.45	5.55	3.5	4.85	5.6	5.85	6.35	8	2	85434	1	2955	4048	3.04790668711445	0.000485179086124
+Verheyen-2019-1000-347	347	hees	hoarse			8.6	2.85	4.89473684210526	3.1	4.3	5.75	3.4	4	1	26063	21	437	16	0.488113220508249	8.56500834827979E-05
+Verheyen-2019-1000-348	348	heet	hot	HOT	1286	5.8	5.1	5.4	5.6	5.05	5.35	4.85	4	1	25735	19	2080	1770	4.83800275933754	0.000447401281445
+Verheyen-2019-1000-349	349	heftig	violent			9.25	6.45	4.4	5.65	4.8	4.05	4.5	6	2	29909	1	1533	495	0.675666315389669	0.000105276058159
+Verheyen-2019-1000-350	350	heilig	holy	HOLY	1702	6.9	3.7	4.3	3.8	4.2	4.35	4.4	6	2	37963	1	2261	795	1.94541604689625	0.000475107391277
+Verheyen-2019-1000-351	351	heimelijk	secret			11.7368421052632	3.33333333333333	4.27777777777778	3.66666666666667	2.47368421052632	3.63157894736842	2.9	9	3	71185	0	528	88	0.662965099748248	6.64894569042754E-05
+Verheyen-2019-1000-352	352	hel	shrill			6.77777777777778	5.05263157894737	4.3	4.4	3.25	5.4	2.4	3	1	28195	27	210	5	1.31735956946746	0.000165712735608
+Verheyen-2019-1000-353	353	helder	clear			8.15	4.45	4.6	4.85	5.15	5.25	5.5	6	2	84752	5	3425	1174	3.35107277346911	0.000466861292814
+Verheyen-2019-1000-354	354	hellend	rapid			8.6	5.05263157894737	4.05263157894737	4.15789473684211	3.1	4.9	3.9	7	2	86636	1	0	0	0.120749963712958	2.71528988062487E-05
+Verheyen-2019-1000-355	355	hemels	heavenly			7.6	4.95	4.15	4.9	3.25	5.2	6.5	6	2	42208	0	546	432	0.679458982959015	7.93890363467454E-05
+Verheyen-2019-1000-356	356	herhaald	repetitious			7.85	3.95	5	4.5	4	3.6	3.75	8	2	80369	0	1183	0	0.045282663290803	8.14847298787713E-06
+Verheyen-2019-1000-357	357	hersteld	better			7.78947368421053	3.8	4.15789473684211	4	4.2	5.15	5.1	8	2	114427	1	0	0	0.052074220817475	7.18523496694597E-06
+Verheyen-2019-1000-358	358	heus	genuine			9.2	3.89473684210526	2.65	3.9	3.25	1.75	4.5	4	1	14145	14	1285	1589	0.037838667161374	1.64010798158549E-05
+Verheyen-2019-1000-359	359	heuvelachtig	hilly			7.8	4.1	5.25	4.05	2.8	6.55	3.55	12	4	95700	0	61	4	0.189038288872164	1.62611499682197E-05
+Verheyen-2019-1000-360	360	hevig	sharp			7.7	6.25	4.2	6	4.8	4.25	4.3	5	2	19997	2	2191	346	2.3767717117327	0.000386395122059
+Verheyen-2019-1000-361	361	hip	hip			8.85	5.3	3.5	5.05	3.85	4.7	5.4	3	1	3234	13	44	272	1.74155361721254	0.000412418819536
+Verheyen-2019-1000-362	362	hoekig	angular	ANGULAR (SQUAREFORMED)	2757	7.175	4.25	5.3	4.3	2.8	6.45	3.05	6	2	29515	2	205	11	0.304728411522255	9.1185447920649E-05
```

Comment:
hoekig relates to angular, but not necessarily to squareformed (cf. German eckig); not sure about the scope of the concept here